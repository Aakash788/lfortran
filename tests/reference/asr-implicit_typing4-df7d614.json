{
    "basename": "asr-implicit_typing4-df7d614",
    "cmd": "lfortran --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/errors/implicit_typing4.f90",
    "infile_hash": "21dfe09305df396a4254e7b1f818acbac4b158b5731a58805794efda",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-implicit_typing4-df7d614.stderr",
    "stderr_hash": "89ebe735df3463eecff8400e8b99ed5c8193b4277d7aded9f2717241",
    "returncode": 2
}