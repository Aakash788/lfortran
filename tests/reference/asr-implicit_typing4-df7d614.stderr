semantic error: Dummy argument 'matvect' not defined
 --> tests/errors/implicit_typing4.f90:1:1 - 3:4
  |
1 |    subroutine iddr_rsvd(matvect)
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
3 |    end
  | ...^^^^ 
